rest-api/model/bq_base.py

```diff
@@ -309,22 +338,135 @@ class BQView(object):
   sql = None
 
 
+# class BQSession(object):
+#
```

Comment:
did you mean to remove these ?

Reply:
Not at this time.  I want to create a testing API that replicates the current PS API, so BQSession might be used to combine Tanner's BQ query job code with the schema code for that testing API.